xml/admin_gui_dashboard.xml

```diff
@@ -2287,6 +2287,89 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
 &prompt.cephuser;ceph dashboard create-self-signed-cert
 </screen>
    </sect3>
+   <sect3 xml:id="self-sign-certificates-openssl">
+     <title>Self-signed or Trusted Third-party Certificate with OpenSSL</title>
+     <para>
+       OpenSSL is an open-source command line tool that is commonly used
+       to generate private keys, create CSRs, install your SSL/TLS
+       certificate, and identify certificate information. The following
+       instructions illustrate to generate a self-signed or trusted third-party
+       certificate using OpenSSL:
+     </para>
+<!-- Source: https://www.suse.com/support/kb/doc/?id=000019241 -->
+     <procedure>
+       <step>
+         <para>
+           Generate a Private Key:
+         </para>
+<screen>
+openssl genrsa -des3 -out server.key 2048
+</screen>
+         <para>
+           Type the pass phrase to protect the key.
+         </para>
+       </step>
+       <step>
+         <para>
+           Generate a Certificate Signing Request:
+         </para>
+<screen>
+openssl req -new -key server.key -out server.csr
+</screen>
+         <para>
+           Enter the passphrase, and fill in the <literal>Country Name</literal>,
+           <literal>State or Province Name</literal>, <literal>Locality Name</literal>, <literal>Organization Name</literal>,
+           <literal>Organizational Unit Name</literal>, <literal>Common Name</literal>,
+           <literal>Email Address</literal>.
+         </para>
+         <note>
+           <para>
+             The <literal>Common Name</literal>e should be the DNS name of the
+             server. For example, <literal>server.mydomain.com</literal>.
+           </para>
+         </note>
+         <para>
+           When asked for a Challenge password and optional company name,
+           leave it blank.
+         </para>
+       </step>
+       <step>
+         <para>
+           To sign the certificate, select from the following options:
+         </para>
+         <itemizedlist>
+           <listitem>
+             <para>
+               (Trusted 3rd-party Certificate Authority) Send the
+               Certificate-Signing Request (CSR) to the third party for their
+               signing. The following files should be received: Server certificate (public key)
+               and the Intermediate CA and the bundles that chain to the Trusted Root CA.
+             </para>
+           </listitem>
+           <listitem>
+             <para>
+               (Self-signed) Sign the certificate with OpenSSL:
+             </para>
+<screen>
+openssl x509 -req -days 730 -in server.csr -signkey server.key -out server.crt
+</screen>
+             <para>
+               Increase or decrease 730 as needed. This is the number
+               of days the certificate is valid for.
+             </para>
+           </listitem>
+         </itemizedlist>
+       </step>
+       <step>
+         <para>
+           Optional: If needed, create a concatenated PEM file:
+         </para>
```

Comment:
With the `performance` option, the stylesheets create an `(optional)` automatically in front of the paragraph. So it looks like `f. (Optional) If needed, create a concatenated PEM file:`

```suggestion
       <step performance="optional">
         <para>
           If needed, create a concatenated PEM file:
         </para>
```